db/migrations/000005_last_token.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
I added the deployment details to the `r47` deployment ticket: https://jira.cms.gov/browse/BCDA-2735